From 46f016c4b87bb2a594df1c84e4d494bff0cbeaf9 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 5 Feb 2018 12:01:25 -0800
Subject: [PATCH] src: Replace using sys/errno.h with errno.h

sys/errno.h is no longer the right place for this file

error: #warning redirecting incorrect #include <sys/errno.h> to <errno.h> [-Werror=cpp]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Submitted [https://github.com/libyui/libyui-ncurses/pull/63]
 src/NCAskForDirectory.cc | 2 +-
 src/NCAskForFile.cc      | 2 +-
 src/NCFileSelection.h    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/NCAskForDirectory.cc b/src/NCAskForDirectory.cc
index 06ec6ac..ed1a30c 100644
--- a/src/NCAskForDirectory.cc
+++ b/src/NCAskForDirectory.cc
@@ -39,7 +39,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <dirent.h>
-#include <sys/errno.h>
+#include <errno.h>
 
 /*
   Textdomain "ncurses"
diff --git a/src/NCAskForFile.cc b/src/NCAskForFile.cc
index 31fdb39..63a6679 100644
--- a/src/NCAskForFile.cc
+++ b/src/NCAskForFile.cc
@@ -39,7 +39,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <dirent.h>
-#include <sys/errno.h>
+#include <errno.h>
 
 /*
   Textdomain "ncurses"
diff --git a/src/NCFileSelection.h b/src/NCFileSelection.h
index d42311f..514d020 100644
--- a/src/NCFileSelection.h
+++ b/src/NCFileSelection.h
@@ -38,7 +38,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <dirent.h>
-#include <sys/errno.h>
+#include <errno.h>
 
 
 struct NCFileInfo
-- 
2.16.1

